docs/manual/common/concepts/index.toc

```diff
@@ -5,3 +5,4 @@ Technologies:Lagom component technologies
 BuildConcepts:Lagom build philosophy
 Immutable:Immutable Objects
 ES_CQRS:Event Sourcing and CQRS
+ServiceDiscovery: Service Discovery
```

Comment:
This shouldn't be here - and I'm not sure why the build is passing with it there.